component/otelcol/processor/resourcedetection/resourcedetection.go

```diff
@@ -31,9 +32,10 @@ import (
 
 func init() {
 	component.Register(component.Registration{
-		Name:    "otelcol.processor.resourcedetection",
-		Args:    Arguments{},
-		Exports: otelcol.ConsumerExports{},
+		Name:      "otelcol.processor.resourcedetection",
+		Stability: featuregate.StabilityStable,
```

Comment:
@thampiotr The docs say this is beta, not stable: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.processor.resourcedetection/